R/between_within.R

```diff
@@ -0,0 +1,105 @@
+#' Calculation of between-within Degrees of Freedom for One-Dimensional Contrast
+#'
```

Comment:
minor comment: can you change the file names to `between-within.R` and `between-within-classic.R` (only because all other files have the dash instead of the underscore)